packages/nodes-base/nodes/Airtable/Airtable.node.ts

```diff
@@ -555,27 +506,15 @@ export class Airtable implements INodeType {
 		const returnData: INodeExecutionData[] = [];
 		let responseData;
 
-		const version = this.getNode().typeVersion;
-
 		const operation = this.getNodeParameter('operation', 0) as string;
 
-		let application: string;
-		if (version === 2) {
-			application = this.getNodeParameter('applicationRLC', 0, undefined, {
-				extractValue: true,
-			}) as string;
-		} else {
-			application = this.getNodeParameter('application', 0) as string;
-		}
+		const application = this.getNodeParameter('application', 0, undefined, {
+			extractValue: true,
```

Comment:
why is this `extractValue` key necessary? could not we simply check the param type? would be simpler to roll out RL without having to update the code everywhere.

Reply:
We don't know what the param type is at this point. Finding out what type something is can be a bit intense depending on how deep it is or based on if it has `displayOptions` or not. It's behind a flag because this could cause performance regressions and this we decided this was a good compromise.

Reply:
maybe I am lost here, but I followed the code and i am not even sure this is getting used.. we should sync on this.